README.md

```diff
@@ -0,0 +1,14 @@
+# My portfolio
+This project was created for the creation of my portfolio.
+![GitHub Logo](./Edward's-Portfolio.png)
+
+# LANGUAGES USED
+- HTML
+- CSS
+- Linters(Lighthouse, Webhint, Stylelint)
+
+# PREREQUISITES
+- Prior knowledge of HTML and CSS.
+
+# Live Demo Link.
+http://127.0.0.1:5500/Index.html#
```

Comment:
- [ ] **[OPTIONAL]** Change the marked titles to have two #. This way, it will look like this:
# One Hashtag
## Two Hashtags
Only the main title (project name) should have 1 hashtag
- [ ] **[OPTIONAL]** Good job mentioning all of the used tools! But remember, the linters aren't languages, but tools that are built to increase your productivity with given languages!
Try to change the `Languages Used` to `Languages & Tools`

- [ ] **[OPTIONAL]** Good job trying to add the live demo link, but keep in mind that the link that you passed is your local server. If you want to deploy the project (remember: this is not mandatory for this project, but I highly recommend you to do so), [read this article about GitHub Pages](https://developer.mozilla.org/pt-BR/docs/Learn/Common_questions/Using_Github_pages).